### PR TITLE
Add "What's new" item for new GA4 tutorial and sample

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### New guidance and sample: Learn how to use Google Analytics 4 in your Chrome extension {: #guide-ga }
+
+<p class="color-secondary-text type--caption">Posted on <time>June 16, 2023</time></p>
```
Comment (AmySteam) [inline site/en/docs/extensions/whatsnew/index.md:20]:
```suggestion
<p class="color-secondary-text type--caption">Posted on <time>June 20, 2023</time></p>
```

Can you also change the updated date in the metadata?

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ tags:
...
+
+We published an [updated version of our Google Analytics guidance](/docs/extensions/mv3/tut_analytics/) explaining how you can use [Google Analytics 4](https://support.google.com/analytics/answer/10089681) in your Chrome extension. We've also added a working [Google Analytics 4 sample](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.google-analytics) to our Github sample repository. Check out [`google-analytics.js`](https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/functional-samples/tutorial.google-analytics/scripts/google-analytics.js) for all the Google Analytics related code.
```
Comment (AmySteam) [inline site/en/docs/extensions/whatsnew/index.md:22]:
```suggestion
We published an [updated version of our Google Analytics guidance](/docs/extensions/mv3/tut_analytics/) explaining how you can use [Google Analytics 4](https://support.google.com/analytics/answer/10089681) in your Chrome extension. We've also added a working [Google Analytics 4 sample](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.google-analytics) to our GitHub sample repository. Check out [`google-analytics.js`](https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/functional-samples/tutorial.google-analytics/scripts/google-analytics.js) for all the relevant code related to Google Analytics.
```
Minor NIT

**Review by AmySteam — APPROVED** [review]
Thanks @sebastianbenz!

Please update the dates before publishing. I left a NIT, but I'm approving because it's not blocker.

File: site/en/docs/extensions/whatsnew/index.md (hunk 2 above)
Comment (sebastianbenz) [reply site/en/docs/extensions/whatsnew/index.md]: fixed

**Pull request merged**